### PR TITLE
Add color customization with Dualbandwidth support

### PR DESCRIPTION
**Pull request by rll-rll**
This work expands chall8908's work and adds color customization to dualbandwidth mode.
+config update to work on Turris Omnia 2019+

### CODE REVIEW

File: install [hunk 1]
```diff
@@ -5,7 +5,7 @@ chmod 755 /usr/sbin/omnia-led-colors
 cp omnia-led-colors.init.d /etc/init.d/omnia-led-colors
 chmod 755 /etc/init.d/omnia-led-colors
 
-[ -f /etc/config/omnia-led-colors ] || cp omnia-led-colors.config /etc/config/omnia-led-colors
+[ /etc/config/omnia-led-colors -nt omnia-led-colors.config ] || cp omnia-led-colors.config /etc/config/omnia-led-colors
```
Comment (ddrown) [inline install]: I don't want this to overwrite someone's customized configuration, only set the defaults if there's no configuration

**Review by ddrown — CHANGES_REQUESTED** [review]
The rest of this code looks fine, just need a change to the install script

File: install (hunk 1 above)
Comment (rll-rll) [reply install]: Reverted commit 0749d99be438f5d1ee8c3430761312e36cedb9a5

**Comment by ddrown** [comment]
Thanks for the PR!

**Pull request merged**